srcalc/src/main/java/gov/va/med/srcalc/domain/model/Variable.java

```diff
@@ -5,12 +5,26 @@
  */
 public interface Variable
 {
+    /**
+     * The maximum length of a valid variable display name: {@value}
+     */
     public static final int DISPLAY_NAME_MAX = 80;
+    
+    /**
+     * The maximum length of a valid variable key: {@value}
+     */
     public static final int KEY_MAX = 40;
     
     /**
-     * Returns a key which should be unique among all Variables. Ideally should
-     * be alphanumeric, though this isn't enforced.
+     * A regular expression that defines a valid variable key: {@value}
+     * @see #getKey()
```

Comment:
I think there should be a description of the regex being any amount of word characters [a-zA-Z0-9_].